### DISCUSSION TITLE
How to use stylesheet in YARD

### DISCUSSION THREAD

**Discussion by heronshoes** [heronshoes/red_amber#182] [Show and tell]
I am using many `@example` in YARD document of RedAmber, but `code` is shown by proportional fonts by default configuration. How can I do to apply monospace fonts for them?

## Register custom template path to override Template

I registered a template path to `.yardopts`.

```
--template-path doc/yard-templates
```

And added a custom css below to `doc/yard-templates/default/fulldoc/html/css/common.css`.

```
/* Use monospace font for code */
code {
  font-family: "Courier New", Consolas, monospace;
}
```

Result:
<img width="862" alt="example" src="https://user-images.githubusercontent.com/63298319/218287581-1143de93-6997-4a07-a8b9-d4b4b2f6802e.png">

note: The custom templates must be in the same directory structure as builtin YARD templates.

YARD document: https://rubydoc.info/gems/yard/file/docs/Templates.md

**Comment by kou** [comment]
How about upstreaming this?
This will be useful for all projects. :-)
